### ISSUE TITLE
Wrong IP Address Name label for Public IP address in Routed VPC Network

### ISSUE THREAD

**Issue by msinhore** [apache/cloudstack#10122]
##### ISSUE TYPE

 * Improvement Request

##### COMPONENT NAME

~~~
VPC
~~~

##### CLOUDSTACK VERSION

~~~
4.20.0
~~~

##### CONFIGURATION

Advanced Networking

##### SUMMARY

In a Routed VPC Network, the Public IP address associated with the Virtual Router’s north interface is incorrectly labeled as “Source NAT.” This is misleading since the Source NAT service is not applicable in Routed VPC mode.

This IP address functions as the external gateway, enabling communication between the upstream router and instances with public IP addresses inside the VPC. The label should be updated to reflect its actual role, with suggestions such as “External Gateway” or another appropriate name indicating its function for routing upstream traffic.

##### STEPS TO REPRODUCE

~~~
	1.	Create a VPC network with Routed mode enabled.
	2.	Navigate to the “Public IP Addresses” tab.
	3.	Observe the Public IP address labeled as “Source NAT.”
~~~

##### EXPECTED RESULTS

~~~
The IP address should have a label that accurately represents its function, such as “External Gateway.”
~~~

##### ACTUAL RESULTS

~~~
The Public IP address is labeled as “Source NAT,” which is misleading in the context of Routed VPC mode.
~~~

**Comment by iishitahere** [comment]
Hello @msinhore @DaanHoogland @weizhouapache ,

I would like to take up this improvement request. Upon reviewing the issue, I understand that in Routed VPC mode, the Public IP address associated with the Virtual Router's north interface is incorrectly labeled as “Source NAT,” which creates confusion as this service is not applicable.

I plan to update the label to accurately reflect its function, such as “External Gateway,” or another appropriate term that better aligns with its role in facilitating communication between the upstream router and VPC instances with public IP addresses.

Please let me know if there are any specific naming conventions or additional considerations I should take into account while implementing this change.

Thank you for your time and consideration.

Best regards,
Ishita Jaiswal

**Comment by msinhore** [comment]
I agree that the current label, "Source NAT," does not accurately reflect the function of the northbound interface on the Virtual Router in Routed VPC mode.  "External Gateway" is quite appropriate, but I would like to propose additional "WAN IP Address"  or something similar as this  highlights the interface's connection to the external or upstream network, aligning with common networking terminology.

**Comment by nvazquez** [comment]
Hi @iishitahere - is this issue addressed by PR #10060?
